src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md

```diff
@@ -380,8 +380,29 @@ Check out our [curated Web Bluetooth Demos](https://github.com/WebBluetoothCG/de
 
 ## Libraries
 
-- [&lt;platinum-bluetooth>](https://elements.polymer-project.org/elements/platinum-bluetooth?active=platinum-bluetooth-device) is a new set of [Polymer](https://www.polymer-project.org/) elements to discover and communicate with nearby Bluetooth devices based on the Web Bluetooth API.
 - [web-bluetooth-utils](https://www.npmjs.com/package/web-bluetooth-utils) is a npm module that adds some convenience functions to the API.
+- [&lt;platinum-bluetooth>](https://elements.polymer-project.org/elements/platinum-bluetooth?active=platinum-bluetooth-device) is a new set of [Polymer](https://www.polymer-project.org/) elements to discover and communicate with nearby Bluetooth devices based on the Web Bluetooth API. For instance, here's how to read battery level from a nearby bluetooth device advertising Battery service:
+
+```html
```

Comment:
In testing this, I discovered in order for code blocks to be properly highlighted on DevSite, they must be indented using 4 spaces, the three backtick wrap does not work (see [3869](https://github.com/google/WebFundamentals/pull/3869)).

To make this work properly, you'll need to remove the back ticks and likely add a `<div class="clearfix"></div>` before the first code block, and another before the second code block.